Xamarin.Forms.Core/RadioButton.cs

```diff
@@ -39,6 +42,12 @@ public string GroupName
 			set { SetValue(GroupNameProperty, value); }
 		}
 
+		public Color RadioColor
```

Comment:
Should we just call it Color?

Reply:
I’m fine with anything really. I just figured it would make it more clear with RadioColor to what it refers. 
And it’s a bit more consistent with other properties like `BackgroundColor` or `TextColor` etc.

"Radio" might not be the best, but I guess it's called `RadioButton` for a reason. So that thing is called the Radio, right? :P

Reply:
I expected `Color` as well...